EapCommons/src/main/java/de/hpi/unicorn/attributeDependency/AttributeDependency.java

```diff
@@ -24,6 +24,17 @@
 import javax.persistence.Table;
 import javax.persistence.UniqueConstraint;
 
+import javax.persistence.Column;

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this duplicated import. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)